Environmental Sensing/GPS Subsystem.md

```diff
@@ -0,0 +1,247 @@
+# Function of the subsystem:
+
+By incorporating a GPS and compass device into the Autonomous Crawlspace Inspection Robot, it would have the capability to accurately determine its position within the crawlspace. This integration of GPS would enable the robot to generate a map of the crawlspace, displaying its current location and the areas it has explored. By combining GPS and compass data, the robot can plan efficient inspection routes and avoid revisiting previously inspected areas. This optimization ensures a comprehensive inspection while minimizing unnecessary movements. Additionally, by correlating the GPS and compass data with sensor readings or captured images, the collected data can be precisely tagged with location information. 
```

Comment:
I can't review this without understanding what this subsystem is supposed to do.

Tell me only what functionality you expect this to have. Don't write a paragraph. Write a single sentence that says what this system's purpose is. What do you expect it to do?